### PR TITLE
Eileen

### PR DESCRIPTION
**Pull request by EileenHan-972**
## Overview

This PR will..

## Activity Finished

*****inverted the motors, but it didn't do anything

### CODE REVIEW

**Comment by EileenHan-972** [comment]
inverted motors

File: src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 1]
```diff
@@ -26,28 +29,34 @@ public class Drivetrain extends SubsystemBase {
...
+    leftMotor1.setIdleMode(IdleMode.kBrake);
+    rightMotor1.setIdleMode(IdleMode.kBrake);
+
+      
     // TODO 1.1.4: Make motor2s follow motor1s
 
     // TODO 1.2.4: Invert motors if necessary
-
+        leftMotor1.setInverted(true);
+        rightMotor1.setInverted(true);
```
Comment (Kyle-Eldridge) [inline src/main/java/frc/robot/subsystems/Drivetrain.java]: I'm still not sure why this part works. Change leftMotor1 to be not inverted for now so it's the same as everyone else's.

File: src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 2]
```diff
@@ -1,12 +1,15 @@
 package frc.robot.subsystems;
 
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkBase.IdleMode;
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkLowLevel.MotorType;
```
Comment (Kyle-Eldridge) [inline src/main/java/frc/robot/subsystems/Drivetrain.java:9]: Remove any unused imports. They should be underlined in yellow in your IDE.

**Review by Kyle-Eldridge — CHANGES_REQUESTED** [review]
The code is mostly good, but there are 2 very small changes you should make. After you finish these, you can start Activity 2 (we will present it on Friday).

File: src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 3]
```diff
@@ -40,8 +39,8 @@ public Drivetrain() {
     // TODO 1.1.4: Make motor2s follow motor1s
 
     // TODO 1.2.4: Invert motors if necessary
-        leftMotor1.setInverted(true);
-        rightMotor1.setInverted(true);
+        leftMotor1.setInverted(false);
+        rightMotor1.setInverted(false);
```
Comment (Kyle-Eldridge) [inline src/main/java/frc/robot/subsystems/Drivetrain.java:55]: The right motor should be inverted, and the left motor should not be.

**Comment by Kyle-Eldridge** [comment]
I am closing all remaining Activity 1 PRs. However, when you make a new PR when you finish Activity 2, you will still need to make any changes left on this PR before we approve the next one.

**Pull request closed without merging**